assets/src/edit-story/elements/text/output.js

```diff
@@ -47,14 +49,25 @@ function TextOutput({
     fontFamily: generateFontFamily(fontFamily, fontFallback),
     fontWeight: fontWeight ? fontWeight : null,
     lineHeight,
-    letterSpacing: letterSpacing ? letterSpacing + 'em' : null,
-    padding: padding ? padding + '%' : null,
+    letterSpacing: isNaN(letterSpacing) ? null : letterSpacing + 'em',
+    padding: padding
+      ? `${dataToEditorY(padding.vertical, 100)}% ${dataToEditorX(
+          padding.horizontal,
+          100
+        )}%`
+      : null,
     textAlign: textAlign ? textAlign : null,
+    textDecoration,
     whiteSpace: 'pre-wrap',
     ...generatePatternStyles(backgroundColor),
     ...generatePatternStyles(color, 'color'),
   };
 
+  // @todo This logic is temporary and will change with selecting part + marking bold.
+  if (bold) {
```

Comment:
Here and above, why shouldn't we just do `<p style="font-weight: bold">...</p>`?

Reply:
It's mainly a pre-step for later when selecting just part of the text should be possible and we'd probably use the tags anyway.
Also, the fonts track a separate font weight param which is based on what's available for each font. It seemed to be more appropriate to use tags.

No strong preference though, LMK if you think it should use `font-weight`.

Reply:
TBH, I can't tell which one might have issues in one case or the other. Let's go with `<strong>` for now and we'll come back to it when we implement buttons on the text selection.